### ISSUE TITLE
Add to cart is super slow, very high TTFB sometimes reaching to 20 seconds when applying discount codes

### ISSUE THREAD

**Issue by hitinder** [PrestaShop/PrestaShop#19687]
Prestashop v1.7.5.1 becomes extremely slow and takes about 5-10 seconds to add a product to the cart. Wen a coupon/discount code is applied it has reached 25 seconds. Most of the time is taken by TTFB. The slow speed issue starts only for the cart related operations.

**Steps to Reproduce**

1. Go to any category
2. Click on 'Add to cart'
3. Scroll down to 'Click on cart'
4. Apply coupon

**Screenshots**

![image](https://user-images.githubusercontent.com/9841229/84255905-eacd2100-ab1b-11ea-8b6a-4a4a036d5c4a.png)

**Additional information**

* PrestaShop version: 1.7.5.1
* PHP version: 7.2
* Our website www.craftingcrow.com

**Comment by TristanLDD** [comment]
Hi @hitinder, thanks for the report

Unfortunately it seems that your issue is not a PrestaShop's core bug but most likely a server configuration or customization problem. I invite you to contact the support for further investigation. 

Thank you

**Comment by hitinder** [comment]
@TristanLDD  My server is a very high config server. We have only installed a theme. Are you saying that theme can cause this issue?

Top 5 Table Stress
```
912 category_lang  
900 category  
673 category_shop  
193 creativepage_meta  
124 cart_product 
```

Top 3 ObjectModel instances

```
Name Instance Source  
Category    1554 /modules/posmegamenu/posmegamenu.php
CartRule    50 /classes/Cart.php
Carrier 22 /classes/Cart.php
```

```
Querying Time   6119 ms  
Queries 1821   
Memory Peak Usage   47.0 Mb  
Included Files  625 files - 8.66 Mb  
PrestaShop Cache    0.01 Mb  
Global vars 0.38 Mb
```

How do I contact support regarding this?
